tests/test_models.py

```diff
@@ -92,3 +92,26 @@ def test_add_product_in_inventory(self):
         self.assertEqual(product_in_inventory.product_in_inventory_id, 1)
         product_in_inventory = InventoryModel.all()
         self.assertEqual(len(product_in_inventory), 1)
+
+
+    def test_update_a_product_in_inventory(self):
+        """ Update a Product In Inventory """
+        product_in_inventory = _test_create_product_in_inventory(
```

Comment:
looks like i named this backwards. should have been `_create_test_product_in_inventory`. not a huge deal since eventually we will replace with the factory stuff